.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - setup_remote_docker:
           version: 20.10.14
           docker_layer_caching: true
-      - run: ./gradlew docker
+#      - run: ./gradlew docker
```

Comment:
TODO: replace this with new build commands since it [fails in circleci](https://app.circleci.com/pipelines/github/department-of-veterans-affairs/abd-vro/733/workflows/6e859239-6f65-46ee-9987-da3632fc9640/jobs/738).

`./gradlew docker` fails locally b/c of [using full path](https://github.com/department-of-veterans-affairs/abd-vro/pull/235/files?diff=unified&w=0#r957800339)